tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -145,11 +146,11 @@ class VarDictJavaEndToEnd(tumorBam: PathToBam,
     val bias               = new ShellCommand(biasScript.toString) with Pipe[Any,Any]
     val toVcf              = new ShellCommand(vcfScript.toString, "-N", tn, "-E", "-f", minimumAf.toString) with Pipe[Any,Vcf]
     val removeRefEqAltRows = new ShellCommand("awk", "{if ($1 ~ /^#/) print; else if ($4 != $5) print}") with Pipe[Vcf,Vcf]
-    val sortVcf            = new SortVcf(dict=Some(dict))
+    val sortVcf            = new SortVcf(in=tmpVcf, out=out, dict=Some(dict))
 
     bias.requires(Cores(0), Memory("32m"))
     toVcf.requires(Cores(0), Memory("32m"))
 
-    root ==> (vardict | bias | toVcf | removeRefEqAltRows | sortVcf > out).withName(this.getClass.getSimpleName)
+    root ==> (vardict | bias | toVcf | removeRefEqAltRows > tmpVcf).withName("VarDictJava") ==> sortVcf ==> new DeleteFiles(tmpVcf)
```

Comment:
I did this for two reasons:
1. I was getting failures with streaming into SortVcf.  But the exact same commands run with writing to an intermediate file worked just fine.
2. The way the SortVcf task works, if it's output is `Io.Stdout` it won't create an index for you, and I want an index.